src/tools/rbd/action/Group.cc

```diff
@@ -777,9 +765,9 @@ int execute_group_snap_list(const po::variables_map &vm,
   }
 
   for (const auto& snap : snaps) {
-    auto state_string = get_group_snap_state_name(snap.state);
+    auto state_string = utils::get_group_snap_state_name(snap.state);
     auto type_string = get_group_snap_namespace_name(snap.namespace_type);
-
+    auto copy_status = "not copied";
```

Comment:
I don't see a reason to have this variable.  The only place where `copy_status` is used could just push into the stream:
```
if (!mirror_snap.complete) {
  oss << "not ";
}
oss << "copied";
```